index.html

```diff
@@ -1162,25 +1156,24 @@ <h4>
   <section> <h3>Save and restore game progress with another device</h3>
     <p>
       Filtering of relevant data sources can be done by the use of
-      the <a>NDEFScanOptions</a>. Below we accept the record identifier URL with
-      "`/mypath/mygame/`" in its path from "`mygame.com`"
-      domain and its subdomains. When we read the data, we immediately update
-      the game progress by issuing a push with a custom NDEF data layout.
+      the <a>NDEFScanOptions</a>. Below we use a custom record identifier so
+      that when we read the data, we immediately update the game progress by
+      issuing a push with a custom NDEF data layout.
     </p>
     <p>
       The example allows reading and pushing to both peers and tags,
       whichever one is tapped first.
     </p>
     <pre class="example">
       const reader = new NDEFReader();
-      await reader.scan({ id: "https://mygame.com/mypath/mygame" });
+      await reader.scan({ id: "my-game-progress" });
```

Comment:
It is not excluded someone could use a URL as id, though.

Reply:
I've edited example description at 7c2d2c1